homeassistant/components/plex/media_browser.py

```diff
@@ -270,52 +343,30 @@ def library_section_payload(section):
 def special_library_payload(parent_payload, special_type):
     """Create response payload for special library folders."""
     title = f"{special_type} ({parent_payload.title})"
+    special_library_id = f"{parent_payload.media_content_id}~{special_type}"
     return BrowseMedia(
         title=title,
         media_class=parent_payload.media_class,
-        media_content_id=f"{parent_payload.media_content_id}:{special_type}",
+        media_content_id=special_library_id,
         media_content_type=parent_payload.media_content_type,
         can_play=False,
         can_expand=True,
         children_media_class=parent_payload.children_media_class,
     )
 
 
-def server_payload(plex_server, platform):
-    """Create response payload to describe libraries of the Plex server."""
-    server_info = BrowseMedia(
-        title=plex_server.friendly_name,
-        media_class=MEDIA_CLASS_DIRECTORY,
-        media_content_id=PLEX_URI_SCHEME + plex_server.machine_identifier,
-        media_content_type="server",
-        can_play=False,
-        can_expand=True,
-        children=[],
-        children_media_class=MEDIA_CLASS_DIRECTORY,
-    )
-    if platform != "sonos":
-        server_info.children.append(special_library_payload(server_info, "Recommended"))
-    for library in plex_server.library.sections():
-        if library.type == "photo":
-            continue
-        if library.type != "artist" and platform == "sonos":
-            continue
-        server_info.children.append(library_section_payload(library))
-    server_info.children.append(BrowseMedia(**PLAYLISTS_BROWSE_PAYLOAD))
-    return server_info
-
-
 def hub_payload(hub):
     """Create response payload for a hub."""
     if hasattr(hub, "librarySectionID"):
-        media_content_id = f"{HUB_PREFIX}{hub.librarySectionID}:{hub.hubIdentifier}"
+        media_content_id = f"{hub.librarySectionID}~{hub.hubIdentifier}"
```

Comment:
Why do `~` magic and not use `/`? That way you could have `plex://<server-id>/server/{hub.hubIdentifier}` and create a model that parses such URLs. We do it like that for media source.

Reply:
Okay, this is done in https://github.com/home-assistant/core/pull/68321/commits/3ad1809557699bec1da97a1a158e3cfcea6b1f61. Is that what you had in mind?